phalcon/Mvc/Collection.zep

```diff
@@ -1015,6 +1024,8 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
 
     /**
      * Allows to perform a summatory group for a column in the collection
+     *
+     * @deprecated
```

Comment:
Please don't introduce deprecations in the brand new version

Reply:
Okay, should I delete the methods?

Reply:
should I delete the methods in this case?

Reply:
Yes, I think so. And don't forget to amend change log for _any notable change_, e.g. changed method signature,  method deletion, etc.

Reply:
It works, I'll do it. For the "createIfNotExists" method, is it useful knowing that it is not documented and that the "uniqness" validator replaces this behavior ?

Reply:
You can either add that in the changelog or add it to the documents or even send me a message so that we can add it in the documentation. :)